trace/obfuscation_config.yaml

```diff
@@ -1,9 +1,157 @@
 sensitive_keys:
   - x-access-token
+  - x-support-secret
+  - x-auth-token
+  - x-csrf-token
+  - token
   - encryption_key
   - sso_jwt_key
   - postman_sid
   - api_key
+  - api-key
+  - x-api-key
+  - auth
+  - auth-key
 
 sensitive_value_regexes:
-  - \bPMAK-[a-f0-9]{24}\b
+  - \bPMAK-[a-f0-9]{24}\b # Unit Test Regex
+  - (?i)https:\/\/creator\.zoho\.com\/api\/[A-Za-z0-9\/\-_\.]+\?authtoken=[A-Za-z0-9]+
+  - \bt1\.[A-Z0-9a-z_-]+[=]{0,2}\.[A-Z0-9a-z_-]{86}[=]{0,2}\b
+  - key"\s*:\s*"username"\s*,\s*"value"\s*:\s*"((live|test)_[a-f0-9]{35})
+  - (?i)https:\/\/[\w-]*\.?zoom\.us\/(j|my)\/[\d\w?=-]+\b
+  - \bb\.AAAAAQ[0-9a-zA-Z_-]{156}\b
+  - (?i)\beyJhbGciOi[a-z0-9_\-\.]{2,1000}\b
+  - \bpypi-AgEIcHlwaS5vcmc[A-Za-z0-9\-_]{50,1000}\b
+  - \bFLWSECK_TEST[a-h0-9]{12}\b
+  - \bnpm_[a-zA-Z0-9]{36}\b
+  - \b[0-9]{15,25}-[a-zA-Z0-9]{20,40}\b
+  - key"\s*:\s*"Authorization"\s*,\s*"value"\s*:\s*"(SSWS [a-zA-Z0-9=_\-]{42})
+  - \bEZAK[a-zA-Z0-9]{54}\b
+  - \b(?:pat|sat)\.[a-zA-Z0-9]{22}\.[a-zA-Z0-9]{24}\.[a-zA-Z0-9]{20}\b
+  - \bico-[a-zA-Z0-9]{32}\b
+  - \bflb_live_[0-9a-zA-Z]{20}\b
+  - \b[0-9a-f]{32}-us[0-9]{1,2}\b
+  - \bdp\.audit\.[a-zA-Z0-9]{40,44}\b
+  - (?i)\bduffel_live_[a-zA-Z0-9_-]{43}\b
+  - \b(amqp|amqps):\/\/[\d\w\:?=-]+\b
+  - \b[A-Za-z0-9]{14}\.atlasv1\.[A-Za-z0-9]{67}\b
+  - (?i)\bsk-ant-api[0-9]{2}-[0-9a-z\-\_]{95}\b
+  - \bdp\.pt\.[a-zA-Z0-9]{40,44}\b
+  - \bAQVN[A-Za-z0-9_\-]{35,38}\b
+  - (?i)\bsk_live_[0-9a-z]{24}\b
+  - '[-]{5}BEGIN EC PRIVATE KEY[-]{5}([\s\S]{128,}?)[-]{5}END EC PRIVATE KEY[-]{5}'
+  - \bhttps:\/\/[\w-]*\.?alchemyapi\.io\/v2\/[\d\w?=-]+\b
+  - \bNRBR-[a-fA-F0-9]{19}\b
+  - \b\d{15,16}(?:\||%)[0-9a-zA-Z_-]{27,40}\b
+  - \bpscale_tkn_[A-Za-z0-9_]{43}\b
+  - \btfp_[0-9A-Za-z-_]{59}\b
+  - \bhttps:\/\/discord\.com\/api\/webhooks\/([0-9]{18,20})\/([0-9a-zA-Z_-]+)\b
+  - (?i)\blin_api_[a-zA-Z0-9]{40}\b
+  - \bdp\.sa\.[a-zA-Z0-9]{40,44}\b
+  - \bdnkey-[a-zA-Z0-9=_\-]{26}-[a-zA-Z0-9=_\-]{52}\b
+  - \b(pk|dk)_(prod|test)_[a-zA-Z0-9]{28}\b
+  - \bglsa_[A-Za-z0-9]{32}_[A-Fa-f0-9]{8}\b
+  - (?i)\bhttps:\/\/api\.hubapi\.com\/webhooks\/v1\/[a-zA-Z0-9]+\/
+  - \bhttps://[a-f0-9]{8}:[a-f0-9]{8}@(?:gems\.contribsys\.com|enterprise\.contribsys\.com)
+  - Bearer xoxe.xox[bp]-\d-[a-zA-Z0-9]{163,166}
+  - \bPMAK-[a-f0-9]{24}-[a-f0-9]{34}\b
+  - \bSK[A-Fa-f0-9]{32}\b
+  - (?i)\bshpat_[a-fA-F0-9]{32}\b
+  - (?i)\bshppa_[a-fA-F0-9]{32}\b
+  - (?i)\bfigd_[0-9a-z_-]{40}\b
+  - \bp8e\-[a-zA-Z0-9\-]{32}\b
+  - Bearer xapp-\d-[A-Z0-9]+-\d+-[a-z0-9]+
+  - (?i)[0-9]+-[0-9a-z_]{32}\.apps\.googleusercontent\.com
+  - (?i)https:\/\/(?:www.)?hooks\.zapier\.com\/hooks\/catch\/[a-z0-9]+\/[a-z0-9]+\/
+  - \b(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}\b
+  - \brzp_live_[0-9a-zA-Z-_]+\b
+  - (?i)\bpk_[0-9a-z]{34}\b
+  - (?i)\bshippo_test_[a-fA-F0-9]{40}\b
+  - ^.{15,1000}$
```

Comment:
This one needs to be removed, we cannot skip all strings that are 15 characters or more.  Obviously some context has been lost.